x/oracle/client/cli/tx.go

```diff
@@ -33,12 +34,17 @@ func GetTxCmd() *cobra.Command {
 // CmdPrevote is the CLI command for sending a Prevote message
 func CmdPrevote() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "prevote [validator] [hash]",
+		Use:   "prevote [validator] [hash] [roundId]",
 		Short: "Broadcast message prevote",
 		Args:  cobra.ExactArgs(2),
```

Comment:
oh actually this needs to be 
```suggestion
		Args:  cobra.ExactArgs(3),
```

right?

Reply:
definitely 😅